### PR TITLE
CM-335: Fix fips-check-image-scan step, refactor ci config for fips o/cert-manager-operator

### PR DESCRIPTION
**Pull request by swghosh**
**`fips-check-image-scan` step**
- Fixes dependencies for SCAN_IMAGE

**[o/cert-manager-operator]**
- Refactor CI config to include and use that step
- Switch from UBI -> ocp base-rhel9 base image, fyr: https://github.com/openshift/check-payload/pull/175/files#diff-9eedccbc831b5f4905cade33d8c75f628c2777e7effff97a9099b3e0d9c31c91

### CODE REVIEW

**Comment by swghosh** [comment]
/cc @rphillips
PTAL at e3c339ef7b350bc231901d8e7ca19c46d154733f (this is a follow-up over https://github.com/openshift/release/pull/52579, as the `dependencies` / `env` stanza in the step was incorrect)

**Comment by bharath-b-rh** [comment]
/pj-rehearse pull-ci-openshift-cert-manager-operator-cert-manager-1.14-fips-image-scan

**Comment by bharath-b-rh** [comment]
/pj-rehearse pull-ci-openshift-cert-manager-operator-cert-manager-1.14-e2e-operator

**Comment by swghosh** [comment]
/cc @TrilokGeer 
PTAL whenever you can, need your review/approval to merge this, TIA!

File: ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.14.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 base_images:
+  base-rhel9:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
```
Comment (TrilokGeer) [inline ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.14.yaml:5]: Needs to be validated for the rhel els image.

**Pull request merged**